docs/extend/extend-apiml/api-mediation-routing.md

```diff
@@ -0,0 +1,94 @@
+# API Mediation Layer routing
+
+As an application developer, route your service to ...
+There are two ways to route to the API Mediation Layer:
```

Comment:
Isn't something missing here?

Reply:
Yeah, it was added by Andrew as he is trying to find a good introduction sentence